pages/contribute.md

```diff
@@ -3,24 +3,119 @@ layout: default
+You don't have to be a member of the Auto-WCAG Community Group to contribute by [giving feedback on rules](#give-feedback-on-rules) or [suggesting new rules](#suggest-new-rules).
 
-## Become a Auto-WCAG Member
+If you want to get even more out of your involvement in our rule creation project, you can officially [join the Auto-WCAG Community Group](#join-the-Auto-WCAG-Community-Group) to get on the mailing list and participate in monthly teleconferences.
 
-The best way to stay informed about the activities of Auto-WCAG is to become a member. By becoming a member you will be added to the mailing list, and you will receive all the latest updates.
+## Overview of how you can contribute
 
-1. To join the Auto-WCAG Community, you need a W3C Account.
-   If you don't already have one, you can get it here:
-   [https://www.w3.org/accounts/request].
-2. Go to **[Join the Auto-WcAG community group now!](https://www.w3.org/community/wp-login.php?redirect_to=%2Fcommunity%2Fauto-wcag%2Fjoin)** and join the group
-3. If you work for a W3C Member organization, you will need to request approval.
-   This request will be sent to your organization's representative in the W3C's
-   Advisory Committee.
+* [Give feedback on rules](#Give-feedback-on-rules)
+    * [Feedback for ideas and early drafts (Issues on Github)](#)
+    * [Feedback and reviews for draft rules (Pull Requests on Github)](#Feedback-for-ideas-and-early-drafts-(Issues-on-Github))
+    * [Feedback/corrections for published rules](#Feedback/corrections-for-published-rules)
+* [Suggest new rules](#Suggest-new-rules)
+    * [Submit an idea for discussion (Issue on Github)](#Submit-an-idea-for-discussion-(Issue-on-Github))
+    * [Submit a draft for a rule (Pull Request on Github)](#Submit-a-draft-for-a-rule-(Pull-Request-on-Github)
+* [Join the Auto-WCAG Community Group](#Join-the-Auto-WCAG-Community-Group)
+    * [Mailing list keeps you up to date](#Mailing-list-keeps-you-up-to-date)
+    * [Participate in our monthly calls](#Participate-in our-monthly-calls)
+    * [How to join the Auto-WCAG Community Group](#How-to-join-the-Auto-WCAG-Community-Group)
 
-## Participate in our monthly calls
+## Give feedback on rules
 
-Auto-WCAG has conference calls every month (usually) on the second Thursdays at 16:00 Central European Time. Participating in these calls is the easiest way to become an active contributor. The invitation and agenda for these meetings is sent out through the mailing list a few days in advance.
+Anyone can provide feedback on rules on our open [Auto-WCAG Github repository](https://github.com/auto-wcag/auto-wcag), so you don't even have to be a member of the Auto-WCAG Community Group to contribute in this way.
```

Comment:
`Anyone` is not very inviting, perhaps keep it simple like.

`Feedback on rules is encouraged`.

Something you forgot to say earlier, is an explanation of why people should watch the repository, so they can get notifications on changes happening.

Reply:
Fixed